docs/protocol/ISM/third-party-ISMs/arbitrum-orbit-ISM.mdx

```diff
@@ -0,0 +1,2 @@
+# Arbitrum Orbit ISM
```

Comment:
For the `OP Stack` and `Polygon PoS`, where we have a hook + ISM setup, we've added a detailed explanation of how the hooks work in references. Do we want to do the same for `Arbitrum Orbit` and `LayerZero V2`